src/components/Accordion/Accordion.jsx

```diff
@@ -0,0 +1,106 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass }  from '../../util/component-types';
+
+import ExpanderPanel from '../ExpanderPanel/ExpanderPanel';
+
+import * as reducers from '../Accordion/Accordion.reducers';
+
+const cx = lucidClassNames.bind('&-Accordion');
+
+const {
+	func,
+	array,
+	object,
+	number,
+	string,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["layout"], "madeFrom": ["ExpanderPanel"]}
+ *
+ * This is a container that renders panels from an array of items
+ * and controls its expansion/retraction.
+ */
+const Accordion = createClass({
+	displayName: 'Accordion',
+
+	reducers,
+
+	propTypes: {
+		/**
+		 * An array of the items to be rendered on accordion
+		 */
+		items: array,
+
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element.
+		 */
+		className: string,
+
+		/**
+		 * Indicates which item is expanded.
+		 */
+		index: number,
+
+		/**
+		 * Called when the user clicks on the component's header of an item.
+		 *
+		 * Signature: `(itemIndex, { event, props }) => {}`
+		 */
+		onChange: func,
+
+		/**
+		 * Passed through to the root element.
+		 */
+		style: object,
+	},
+
+	getDefaultProps() {
+		return {
+			items: [],
```

Comment:
Rather than using an array of objects here, we use a child composition pattern that allows the code to look something like below. Take a look at how `Tabs` is implemented to get an idea for how it should look.

```
<Accordion>
  <Accordion.Item Header='stuff'>content goes here</Accordion.Item>
  <Accordion.Item Header='another>content here</Accordion.Item>
</Accordion>
```